include/Effect.h

```diff
@@ -227,11 +213,11 @@ class LMMS_EXPORT Effect : public Plugin
 
 private:
 	/**
-		If the setting "Keep effects running even without input" is disabled,
-		after "decay" ms of a signal below "gate", the effect is turned off
-		and won't be processed again until it receives new audio input
-	*/
-	void checkGate(double outSum);
+	 * If auto-quit is enabled ("Keep effects running even without input" setting is disabled),
+	 * after "decay" ms of a signal below the silence threshold, the effect is
+	 * turned off and won't be processed again until it receives new audio input.
+	 */
+	void handleAutoQuit(std::span<const SampleFrame> output);
```

Comment:
I think the documentation could better describe what this function is doing. Maybe make the name of the function more relevant to what it does and not how it is being used as well.

Reply:
After this PR is merged, I plan to move the silent buffer checking to `AudioPorts::Router` in #7459, so this method will only increment/reset the silent buffer count and activate auto-quit when needed. The current name is already fine IMO, but it will become more fitting in the near future.